.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,82 @@
+<!-- markdownlint-disable first-line-heading -->
+<!-- Thank you for your contribution! -->
+<!-- First time contributors: Take a moment to review https://als.rtfd.io/en/latest/contributing/guidelines! -->
+
+❓ **What kind of change does this PR introduce?**
+
+* [ ] 🐞 bug fix
```

Comment:
What is the real benefit of having this question?

Reply:
It's nice to see the intent/perception of the submitter. There's still no forms for PRs.